docs/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>Maybe it's worth to mention what the inputs are, as this is an important part of the RandomVariable API. I am not totally convinced though...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>By "inputs" you mean <code>RandomGeneratorSharedVariable</code> and / or what else? </p>

Reply:
<p>Yes</p>

Reply:
Added some comments from the original notebook in https://github.com/pymc-devs/pymc/pull/5721/commits/b0781cf835d30257a8458bac80c76db3556936a6

Reply:
<p>Added some comments from the original notebook in https://github.com/pymc-devs/pymc/pull/5721/commits/b0781cf835d30257a8458bac80c76db3556936a6</p>